build-tools/automation/azure-pipelines.yaml

```diff
@@ -571,7 +561,56 @@ stages:
       parameters:
         artifactName: mac-msbuild-test-results
 
- # Check - "Xamarin.Android (Test MSBuild With Emulator Mac)"
+  # Check - "Xamarin.Android (Test MSBuild Win)"
+  - job: win_msbuild_tests
+    displayName: MSBuild Win
```

Comment:
Because of [this](https://github.com/xamarin/xamarin-android/pull/3643#issuecomment-532689933), should this `displayName` make it obvious that this is testing *macOS*-built packages on Windows, and not Windows-built packages on Windows?

Perhaps we want: `MSBuild Win (macOS .vsix)`?  Or perhaps we don't need to worry that much?

Reply:
Maybe instead of having jobs/checks named `Test $foo Mac` and `Test $foo Win` we could just specify the installer extension, which also makes it clear the commercial artifacts are being tested. For example `Test MSBuild With Emulator - .vsix`?

I agree with keeping the Windows stage around, and in doing so we can perhaps safely assume that every job in the `Test` stage is running against macOS produced installers.

Reply:
Putting the installer extension in a the job looked a bit funny to me, I went with `Test $foo - Windows` and `Test $foo - macOS` for now.